docs/HOWTO-bn.md

```diff
@@ -1,34 +1,36 @@
-# How-To at a glance
+# তথ্য এক নজরে
 
 <div align="right" markdown="1">
 
 *[অন্য ভাষায় এটা পড়ুন](README.md#translations)*
 
 </div>
 
-**`Free-Programming-Books` রিপজিটরি তে স্বাগতম!**
+**`Free-Programming-Books`-এ স্বাগতম!**
 
-আমরা নবাগত কন্ট্রিবিউটরস্ দের স্বাগতম জানাই; এমনকি যারা GitHub এ প্রথম কোন Pull Request (PR) তৈরি করেছেন। যদি আপনি তাদের একজন হয়ে থাকেন তাহলে নিচের রিসোর্স গুলো আপনার কাজে লাগতে পারেঃ
+আমরা নতুন যোগদানকারীদের স্বাগত জানাই; এমনকি যারা তাদের প্রথম পুল রিকোয়েস্ট (PR) টি গিটহাবে প্রেরণা নিয়ে তৈরি করছেন। আপনি যদি একজন তাদের মধ্যে থাকেন, তবে নিম্নলিখিত সম্পদগুলি সাহায্য করতে পারে:
 
-* [পুল রিকোয়েস্ট কি?](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(in english)*
-* [কিভাবে পুল রিকোয়েস্ট দিব](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) *(in english)*
-* [GitHub হ্যালো ওয়ার্ল্ড](https://docs.github.com/en/get-started/quickstart/hello-world) *(in english)*
-* [YouTube - নতুনদের জন্য GitHub](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*
-* [YouTube - কিভাবে একটি GitHub রিপোজিটরি ফোর্ক করবেন এবং পুল রিকোয়েস্ট সাবমিট করবেন](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(in english)*
-* [YouTube - মার্কডাউন ক্র্যাশ কোর্স](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in english)*
+* [পুল রিকোয়েস্ট সম্পর্কে](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(in english)*
+* [পুল রিকোয়েস্ট তৈরি করা](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) *(in english)*
+* [গিটহাব হ্যালো ওয়ার্ল্ড](https://docs.github.com/en/get-started/quickstart/hello-world) *(in english)*
+* [ইউটিউব - গিটহাব টিউটোরিয়াল প্রাথমিকদের জন্য](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*
```

Comment:
* [ইউটিউব - গিটহাব টিউটোরিয়াল নতুনদের জন্য](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in english)*